package.json

```diff
@@ -28,7 +28,7 @@
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx}": [
-      "eslint --fix",
+      "eslint --fix --quiet",
```

Comment:
warnings are not actionable, so no need to show them during commit and in CI logs because they clatter output and with big amount of warnings, it's very hard to find the actual error, especially in GHA logs